src/data/regex-quiz.ts

```diff
@@ -1,5 +1,15 @@
 const regexQuiz = [
+    {
+    Question: "Consider a regular expression pattern: `^([a-zA-Z]+)\d{2,}([!@#$%^&*]+)$`.Which of the following strings will match this pattern?",
+    Answer: "`Hello123@`",
+    Distractor1: "Abc123!@",
+    Distractor2: "XYZ567@#$",
+    Distractor3: "Test12!@#$",
+    Explanation: "Abc123!@, matches this pattern because it starts with alphabetical characters, followed by at least two digits, and ends with special characters !@.",
+    Link: "https://www.freecodecamp.org/news/regular-expressions-for-beginners/",
+  },
```

Comment:
For regex questions, please avoid creating questions with code examples because they don't always render well in the quiz where these questions are used.
Sometimes there are issues with the `\` like in this case with the failing lint test.

Please come up with another question